gulpfile.js

```diff
@@ -8,7 +8,7 @@ var gutil = require('gulp-util');
 var connect = require('gulp-connect');
 var webpack = require('webpack');
 var webpackStream = require('webpack-stream');
-var uglify = require('gulp-uglify');
+var terset = require('gulp-terser');
```

Comment:
I think there is typo. `var terset` should be `var terser`

Reply:
oops. Just fixed this